src/devices/Ssd1331/Ssd1331/Ssd1331.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>

```

Comment:
see this file as an example, just use the same: https://github.com/dotnet/iot/blob/main/src/devices/Ssd1351/Ssd1351.csproj

You don't need to specify anything that is already present. So start with copy/pasting the file and then see if anything else is missing.

We do use project reference, not the nuget for System.Device.Gpio but again, all is transparent